barretenberg/cpp/src/barretenberg/stdlib/hash/blake3s/blake3s.test.cpp

```diff
@@ -135,3 +135,27 @@ TEST(stdlib_blake3s, test_multiple_sized_blocks)
         EXPECT_EQ(proof_result, true);
     }
 }
+
+// Edge case that caused addition overflow issues in Blake
```

Comment:
I suggest to write a detailed comment on what the issue was, and what your fix was. Helps anyone new looking at the code understand the context of the fix.